pkg/tuya/api.go

```diff
@@ -162,9 +163,13 @@ func NewTuyaClient(openAPIURL string, deviceID string, uid string, clientID stri
 	}
 
 	if useRTSP {
-		if err := client.GetRTSP(); err != nil {
+		if err := client.GetStreamUrl("rtsp"); err != nil {
 			return nil, fmt.Errorf("failed to get RTSP URL: %w", err)
 		}
+	} else if useHLS {
+		if err := client.GetStreamUrl("hls"); err != nil {
+			return nil, fmt.Errorf("failed to get HLS URL: %w", err)
+		}
```

Comment:
I think this fallback mechanism is not useful. In my tests, the RTSP stream URL is always returned, but the stream itself doesn't work when played.

Reply:
Oops, never mind. I misread the code.